package rut.miit.testingsystem.exception;

public class TestNotFoundException extends RuntimeException {
}
